folium/folium.py

```diff
@@ -587,6 +587,50 @@ def click_for_marker(self, popup=None):
         click_str = click_temp.render({'popup': popup_txt})
         self.template_vars.update({'click_pop': click_str})
 
+    def fit_bounds(self, bounds, padding_top_left=None,
+            padding_bottom_right=None, padding=None, max_zoom=None):
+        """Fit the map to contain a bounding box with the maximum zoom level possible.
+
+        Parameters
+        ----------
+        bounds: list of (latitude, longitude) points
+            Bounding box specified as two points [southwest, northeast]
+        padding_top_left: (x, y) point, default None
+            Padding in the top left corner. Useful if some elements in
+            the corner, such as controls, might obscure objects you're zooming
+            to.
+        padding_bottom_right: (x, y) point, default None
+            Padding in the bottom right corner.
+        padding: (x, y) point, default None
+            Equivalent to seting both top left and bottom right padding to
+            the same value.
+        max_zoom: int, default None
+            Maximum zoom to be used.
+
+        Example
+        -------
+        >>>map.fit_bounds([[52.193636, -2.221575], [52.636878, -1.139759]])
```

Comment:
I didn't introduce a space because the rest of the file follows the no-space convention. (I personally prefer the space too)